bin/node/runtime/src/lib.rs

```diff
@@ -757,6 +765,31 @@ impl pallet_bags_list::Config<VoterBagsListInstance> for Runtime {
 	type WeightInfo = pallet_bags_list::weights::SubstrateWeight<Runtime>;
 }
 
+/// A special VoterList instance used to test the StakeTracker.
```

Comment:
impl blocks should not be documented. Trait definitions should be. If you want to put some doc here, it should start with `//`.

Reply:
This is just a rough draft so far, I basically put some random docs here in there to define the intention. The documentation will be refined! Thanks for looking into this.